src/overlays/actors/ovl_Obj_Kibako/z_obj_kibako.c

```diff
@@ -219,14 +219,15 @@ void ObjKibako_Idle(ObjKibako* this, PlayState* play) {
 void ObjKibako_SetupHeld(ObjKibako* this) {
     this->actionFunc = ObjKibako_Held;
     this->actor.room = -1;
-    func_8002F7DC(&this->actor, NA_SE_PL_PULL_UP_WOODBOX);
+    //! @bug: This is an unsafe cast, although the sfx will still play
+    Player_PlaySfx((Player*)&this->actor, NA_SE_PL_PULL_UP_WOODBOX);
```

Comment:
I wasnt sure about a bug tag at first, but the moment a varaible outside of the base actor struct is accessed, it will be a problem
mm does add those kind of accesses, and its fair to a say modders could too.

so, it makes sense